docs/framework/guides/deep-dive/observables.md

```diff
@@ -0,0 +1,303 @@
+
+```js
+const command = new Command( 'bold' );
+
+command.on( 'change:value', ( evt, propertyName, newValue, oldValue ) => {
+    console.log(
+        `${ propertyName } has changed from ${ oldValue } to ${ newValue }`
+    );
+} )
+
+command.value = true; // -> 'value has changed from undefined to true'
+command.value = false; // -> 'value has changed from true to false'
+
+command.name = 'italic'; // -> changing a regular property fires no event
+```
+
+During its life cycle, an instance of the `Command` can be enabled and disabled many times just as its `value` can change very often and different parts of the application will certainly be interested in that state.
+
+For instance, some commands can be represented by a button, which should be able to figure out its look ("pushed", disabled, etc.) as soon as possible. Using observable properties makes it a lot easier because all the button must know about its command is the names of properties to listen to apply changes instantly.
+
+Additionally, as the number of observable properties increases, you can save yourself the hassle of creating and maintaining multiple `command.on( 'change:property', () => { ... } )` listeners by sharing command's state with the button using [bound properties](#property-bindings), which are the key topic of the next chapter.
+
+## Property bindings
+
+One observable can also propagate its state (or part of it) to another observable to simplify the code and avoid numerous `change:property` event listeners. First, make sure both objects (classes) mix the {@link module:utils/observablemixin~ObservableMixin}, then use the {@link module:utils/observablemixin~ObservableMixin#bind `bind()`} method to create the binding.
+
+### Simple bindings
+
+Let's consider two objects: a `command` and a corresponding `button` (both {@link module:utils/observablemixin~Observable}).
+
+```js
+const command = new Command( 'bold' );
+const command = new Button();
+```
+
+Any "decent" button must update its look when the command becomes disabled. A simple property binding doing that could look as follows:
+
+```js
+button.bind( 'isEnabled' ).to( command );
+```
+
+After that:
+
+* `button.isEnabled` **instantly equals** `command.isEnabled`,
+* whenever `command.isEnabled` changes, `button.isEnabled` will immediately reflect its value.
+
+Note that `command.isEnabled` **must** be defined using the `set()` method for the binding to be dynamic – we did that in the [previous chapter](#making-properties-observable). The `button.isEnabled` property does not need to exist prior to the `bind()` call and in such case, it will be created on demand. If the `button.isEnabled` property is already observable, don't worry: binding it to the command will do no harm.
```

Comment:
```suggestion
Note that `command.isEnabled` **must** be defined using the `set()` method for the binding to be dynamic &mdash; we did that in the [previous chapter](#making-properties-observable). The `button.isEnabled` property does not need to exist prior to the `bind()` call and in such case, it will be created on demand. If the `button.isEnabled` property is already observable, don't worry: binding it to the command will do no harm.
```